com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/tester.py

```diff
@@ -97,8 +98,14 @@ def setup_distributed(test):
         test.username = os.getenv("STREAMS_USERNAME", "streamsadmin")
         test.password = os.getenv("STREAMS_PASSWORD", "passw0rd")
 
+        # Disable SSL verification
+        from requests.packages import urllib3
+        urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
```

Comment:
I had originally put this in to silence the warning:
```
connectionpool.py:734: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.org/en/latest/security.html
  InsecureRequestWarning)
```

But I don't seem to be getting it anymore even though I remove the code... so I'm going to leave it out for now.